src/_pyinstaller_hooks_contrib/tests/test_libraries.py

```diff
@@ -634,3 +634,12 @@ def test_mariadb(pyi_builder):
     pyi_builder.test_source("""
         import mariadb
         """)
+
+
+@importorskip('dash_uploader')
+def test_dash_uploader(pyi_builder):
+    pyi_builder.test_source("""
+        import dash
+        import dash_html_components
```

Comment:
What's the motivation behind these two lines?

Reply:
That was a mistake. I will correct it now.